cfg/SurfTimer/server_settings.cfg

```diff
@@ -17,7 +17,8 @@ sv_deadtalk 1
 sv_full_alltalk 1
 
 // Movement Settings
-sv_airaccelerate 150
+// some pussy shit = sv_airaccelerate 150
+sv_airaccelerate 2000
```

Comment:
Yeah no, we're keeping it at 150 - going to standard settings here

Reply:
Ye that's my fault oopsie